lib/core/src/lib/form/components/widgets/core/form.model.ts

```diff
@@ -100,7 +100,8 @@ export class FormModel implements ProcessFormModel {
         formValues?: FormValues,
         readOnly: boolean = false,
         protected formService?: FormValidationService,
-        enableFixedSpace?: boolean
+        enableFixedSpace?: boolean,
+        injectedFieldValidators?: FormFieldValidator[]
```

Comment:
maybe worth calling "extraValidators" or something generic, as the form class is not "aware" of the DI